docker/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script creates the necessary files before starting Celestia-appd
+
+# only create the priv_validator_state.json if it doesn't exist and the command is start
+if [[ $3 == "start" && ! -f $CELESTIA_HOME/data/priv_validator_state.json ]]
```

Comment:
Maybe nit, but there's no guarantee that `$CELESTIA_HOME` doesn't contain spaces or special characters. You can surround it in quotes to avoid things breaking if it does.

Reply:
good catch :+1: